types/interface-connection/index.d.ts

```diff
@@ -5,11 +5,22 @@
 
 /// <reference types="multiaddr"/>
 
+declare interface Stream {
+    source: Function;
+    sink: Function;
```

Comment:
```
  sink(source: Iterable<unknown>);
  source: Record<string, unknown>;
```
these might be better types

Reply:
I'll see if I have other PRs I'll include it,
or I'll include it separately.

Reply:
I think there is also AsyncIterator type, would be nice to typehint it properly